### PR TITLE
docs(site): improved faq and kopiaignore section 

### LINKED ISSUE #2681: Kopia shows Zero bytes for the snapshot size if the latest snapshot is incomplete
Screenshot from 2023-01-06 01-41-32
Screenshot from 2023-01-06 01-41-44

### PR DESCRIPTION
**Pull request by lupusA**
The PR comes with the following changes:

- Added the documentation on kopiaignore to the FAQs
- Added the FAQs to the main menu
- Added a link the kopiaignore documentation in the "features" section
- Added a paragraph on incomplete snapshots in the FAQs. 

This PR closes #2681

Feedback is welcomed. 

Cheers,

### CODE REVIEW

File: site/content/docs/Advanced/Kopiaignore/_index.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ weight: 40
 
 Users may want to exclude folders and files not to be saved within the repository when creating snapshots. The benefits of omitting unnecessary files and folders are smaller and faster snapshots while saving only essential data. 
 
-Kopia features `pattern-based` ignore rules to omit folders and files from snapshots. While scanning directories and their content, Kopia looks explicitly for files that contain such patterns. 
-If such a file is placed within a directory, `Kopia` omits files and folders `matching` the pattern.
+Kopia features `pattern-based` ignore rules to omit folders and files from snapshots. While scanning directories and their content, Kopia looks explicitly for files that contain such rules.
```
Comment (julio-lopez) [inline site/content/docs/Advanced/Kopiaignore/_index.md:11]:
```suggestion
Kopia uses `pattern-based` ignore rules to omit folders and files from snapshots. While scanning directories and their content, Kopia looks explicitly for files that contain such rules. 
```

File: site/content/docs/Advanced/Kopiaignore/_index.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ weight: 40
 
 Users may want to exclude folders and files not to be saved within the repository when creating snapshots. The benefits of omitting unnecessary files and folders are smaller and faster snapshots while saving only essential data. 
 
-Kopia features `pattern-based` ignore rules to omit folders and files from snapshots. While scanning directories and their content, Kopia looks explicitly for files that contain such patterns. 
-If such a file is placed within a directory, `Kopia` omits files and folders `matching` the pattern.
+Kopia features `pattern-based` ignore rules to omit folders and files from snapshots. While scanning directories and their content, Kopia looks explicitly for files that contain such rules. 
+If such a file is placed within a directory, `Kopia` omits files and folders `matching` the rule.
```
Comment (julio-lopez) [inline site/content/docs/Advanced/Kopiaignore/_index.md:12]: plural? there may be more than one rule in an ignore file, right?
I guess my confusion here is regarding terminology
What is a rule vs what is a pattern?
```suggestion
If such a file is placed within a directory, `Kopia` omits files and folders `matching` the rules.
```

File: site/content/docs/FAQs/_index.md [hunk 3]
```diff
@@ -35,7 +37,7 @@ See the [repository help docs](../repositories) for more information.
 
 #### What is a Policy?
 
-A `policy` is a set of rules that tells Kopia how to create/manage snapshots; this includes features such as [compression, snapshot retention, and scheduling when to take automatically snapshots](../features#policies-control-what-and-how-filesdirectories-are-saved-in-snapshots).
+A `policy` is a set of rules that tells Kopia how to create/manage snapshots; this includes features such as [compression, snapshot retention, and scheduling when to take snapshots automatically](../features#policies-control-what-and-how-filesdirectories-are-saved-in-snapshots).
```
Comment (julio-lopez) [inline site/content/docs/FAQs/_index.md:40]: That is fine, or alternatively:

```suggestion
A `policy` is a set of rules that tells Kopia how to create/manage snapshots; this includes features such as [compression, snapshot retention, and scheduling when to automatically take snapshots](../features#policies-control-what-and-how-filesdirectories-are-saved-in-snapshots).
```

File: site/content/docs/FAQs/_index.md [hunk 4]
```diff
@@ -49,14 +51,20 @@ Files/directories are restored from the snapshots you create. To restore the dat
...
+Encryption is at the `repository` level, and Kopia encrypts all snapshots in all repositories by default. Kopia asks for a password when creating your `repository`. This password is used to encrypt your backups. 
 
 By default, Kopia uses the `AES256-GCM-HMAC-SHA256` encryption algorithm for all repositories, but you can choose `CHACHA20-POLY1305-HMAC-SHA256` if you want to. Picking an encryption algorithm is done when you initially create a `repository`. In `KopiaUI`, to pick the `CHACHA20-POLY1305-HMAC-SHA256` encryption algorithm, you need to click the `Show Advanced Options` button at the screen where you enter your password when creating a new `repository`. For Kopia CLI users, you need to use the `--encryption=CHACHA20-POLY1305-HMAC-SHA256` option when [creating a `repository`](../getting-started/#creating-a-repository) with the [`kopia repository create` command](../reference/command-line/common/#commands-to-manipulate-repository).
 
 Currently, encryption algorithms cannot be changed after a `repository` has been created.
 
+> NOTE Do not forget your repository password. If it is lost, there is no way to recover it or the files and folders within that repository.
```
Comment (julio-lopez) [inline site/content/docs/FAQs/_index.md:66]: WDYT about something along these lines?
```suggestion
> NOTE There is no way to recover it or the files and folders within that repository. Store your repository password in a safe place, such as a password manager, so you can retrieve it later.
```

**Review by julio-lopez — COMMENTED** [review]
LG, a couple of minor suggestions.
PTAL, then we can run this through CI

**Comment by julio-lopez** [comment]
Actually, since this is a docs only change, there is no need to do a full CI run for it.
The rendered docs are at https://deploy-preview-2864--kopia.netlify.app/

File: site/content/docs/FAQs/_index.md (hunk 4 above)
Comment (lupusA) [reply site/content/docs/FAQs/_index.md:66]: That's better, thanks :-)

File: site/content/docs/FAQs/_index.md (hunk 3 above)
Comment (lupusA) [reply site/content/docs/FAQs/_index.md:40]: Thanks for the feedback. I would leave it as it is, i guess.

**Comment by lupusA** [comment]
Hey @julio-lopez, 

thanks for the feedback. I accepted all but one change that was suggested. 
The PR can be merged imo. 

Cheers,

**Pull request merged**